morepath/tests/test_view_directive.py

```diff
@@ -56,7 +55,7 @@ class Core(morepath.App):
     class App(Core):
         pass
 
-    @Core.predicate(generic.view, name='extra', default='DEFAULT',
+    @Core.predicate(morepath.App._view, name='extra', default='DEFAULT',
```

Comment:
This illustrates my previous remark: ``morepath.App._view`` should be public. I used ``morepath.App.get_view`` but I'm fine with anything that does not have a leading underscore.